sql/ordering/_cuthillMckeeOrdering.sql

```diff
@@ -0,0 +1,49 @@
+/*PGR-GNU*****************************************************************
+File: _cuthillMckeeOrdering.sql
+
+Generated with Template by:
+Copyright (c) 2022 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2022 Shobhit Chaurasia
+Mail: 000shobhitchaurasia at gmail.com
+
+------
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+----------------------------
+-- _pgr_cuthillMckeeOrdering
+----------------------------
+
+CREATE FUNCTION _pgr_cuthillMckeeOrdering(
+    edges_sql TEXT,
+    s_vertex INTEGER,
+    OUT r_cuthillMckeeOrdering BIGINT[],
```

Comment:
Why is the prefix needed?

Reply:
Because of the reverse ordering. Basically, the algorithm defined in Boost gives reverse ordering so in pgRouting I will also give the reverse ordering.